### PR TITLE
Added solution with Spring

### PR DESCRIPTION
**Pull request by Nokafan**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -15,17 +16,51 @@
...
+            <version>2.8.0</version>
+        </dependency>
+        <dependency>
+            <groupId>javax.servlet</groupId>
+            <artifactId>javax.servlet-api</artifactId>
+            <version>4.0.1</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>javax.servlet.jsp</groupId>
```
Comment (frostpv) [inline pom.xml]: Мы не используем эту зависимость в проекте

**Review by frostpv — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (Nokafan) [reply pom.xml]: Дякую, виправив.

File: src/main/java/com/dev/cinema/dao/impl/GenericDaoImpl.java [hunk 2]
```diff
@@ -2,20 +2,30 @@
...
-import com.dev.cinema.util.HibernateUtil;
 import lombok.extern.log4j.Log4j;
 import org.hibernate.Session;
+import org.hibernate.SessionFactory;
 import org.hibernate.Transaction;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Repository;
 
 @Log4j
+@Repository
```
Comment (Sofasmile) [inline src/main/java/com/dev/cinema/dao/impl/GenericDaoImpl.java]: should we use Repository here?

Comment (Nokafan) [reply src/main/java/com/dev/cinema/dao/impl/GenericDaoImpl.java]: We have this class as part of other classes, it is not individual so we no need it here.

**Review by Sofasmile — APPROVED** [review]

**Pull request merged**